### ISSUE TITLE
Launch encrypted script does not work

### ISSUE THREAD

**Issue by tineheller** [otris/vscode-janus-debug#185]
When **Upload and Debug Script** is executed on an encrypted script, the script only returns something like that:
```
...
Script-Error: -1
Script-ErrorMsg: JavaScript error: SyntaxError: illegal character. Error was in line 1 of myScript (...)
...
```
After that, VS Code (not the server!) must be restarted.

There are two things to be fixed:
1) The bug is caused by the internally added `debugger;` statement in server and must be fixed there.
2) The problem that the extension cannot handle the problem, and VS Code must be restarted, has to be fixed in the extension.

**Comment by tineheller** [comment]
The fix in server is available in version 5.0d HF2.

**Comment by tineheller** [comment]
The fix in extension is done with 197fb12c61296b4c956f726a8f30055d36d00437.
